lua/entities/gmod_wire_egp/lib/objects/box.lua

```diff
@@ -24,20 +24,11 @@ Obj.DataStreamInfo = function( self )
 	table.Merge( tbl, { angle = self.angle } )
 	return tbl
 end
-function Obj:Contains(point, this)
-	local _, realpos = EGP:GetGlobalPos(this, self.index)
-	local x, y = point.x - realpos.x, point.y - realpos.y
-	
-	local theta = math.rad(realpos.angle + self.angle)
-	if theta ~= 0 then
-		local cos_theta, sin_theta = math.cos(theta), math.sin(theta)
-		x, y =
-			x * cos_theta - y * sin_theta,
-			y * cos_theta + x * sin_theta
-	end
+function Obj:Contains(x, y, egp)
```

Comment:
Sorry to nitpick, but could you make egp the first param, like it is with the other functions?

Reply:
The rest of the code looks much better now, thanks.

Reply:
Yeah I get it